packages/apps/human-app/frontend/src/components/layout/protected/drawer-navigation.tsx

```diff
@@ -72,7 +74,17 @@ export function DrawerNavigation({
         variant="persistent"
       >
         {!isMobile && (
-          <Stack alignItems="flex-start" sx={{ paddingLeft: '60px' }}>
+          <Stack
+            alignItems="flex-start"
+            sx={{ paddingLeft: '60px', cursor: 'pointer' }}
+            onClick={() => {
+              handleMainNavIconClick();
+              // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+              if (isMobile) {
```

Comment:
This component is rendered only for desktop (`!isMobile &&` above), do we need this code block at all?